projects/packages/my-jetpack/_inc/data/utils/get-guessed-site-lifecycle-status.ts

```diff
@@ -1,5 +1,9 @@
-const getGuessedSiteLifecycleStatus = () => {
-	const lifecycleStats = window?.myJetpackInitialState?.lifecycleStats;
+type lifecycleStatsType = Window[ 'myJetpackInitialState' ][ 'lifecycleStats' ];
```

Comment:
should start with uppercase; also, we can just write `Window[ 'myJetpackInitialState' ][ 'lifecycleStats' ]` in the function definition, as it's quite straightforward